tests/managers_regress/models.py

```diff
@@ -147,3 +147,70 @@ class RelationModel(models.Model):
 
     def __str__(self):
         return force_text(self.pk)
+
+
+class BaseModel(models.Model):
```

Comment:
@charettes, how do you recommend to handle these models? should we add `managed=False` to them all since we don't need the tables? Maybe we should create a models module and put them in a separate file too.